Source/DafnyLanguageServer.Test/CodeActions/CodeActionsTest.cs

```diff
@@ -178,5 +295,32 @@ method f() returns (i: int)
           $"Did not find the code action '{expectedTitle}'. Available were:{string.Join(",", otherTitles)}");
       }
     }
+
+    private static List<string> ApplyEdits(TextDocumentEdit? textDocumentEdit, string output) {
+      var inversedEdits = textDocumentEdit.Edits.ToList()
+        .OrderBy(x => -1000 * x.Range.Start.Line - x.Range.Start.Character);
```

Comment:
Would it be possible to do this ordering by lexicographic comparison on tuples, instead? If the code happened to have a very long long, this would miscalculate.

Reply:
I just found how, it's pretty elegant:
```
         .OrderByDescending(x => x.Range.Start.Line)
        .ThenByDescending(x => x.Range.Start.Character);
```

Reply:
Nice!